viewer.py

```diff
@@ -348,7 +348,7 @@ def asset_loop(scheduler):
 
 def setup():
     global HOME, db_conn, browser_bus
-    HOME = getenv('HOME', '/home/pi')
+    HOME = getenv('HOME', 'HOME')
```

Comment:
This one should be changed from `HOME = None` to `user_home_dir = None` right?

Reply:
Well, you can just change it to `HOME = getenv('HOME', False)` (which i think is the default behavior, but to make it more explicit). You then need to have some logic when you call on it, like:

```
If not HOME:
   exit(1)
```

Or similar.

Reply:
Ok, i can set the default, but the part that calls it is this one below, and so that means i would have to edit that as well to exit if there is no home directory?

orig:
```
def setup():
    global HOME, db_conn, browser_bus
    HOME = getenv('HOME')
```

new:
```
def setup():
    global HOME, db_conn, browser_bus
    if not HOME:
        logging.error('No HOME variable')
        exit(1)
    else:
        HOME = getenv('HOME')
```

Reply:
That was more psuedo code, but something like that. @nicomiguelino feel free to chime in as you're in the process of refactor.

Reply:
We can do something like this...


```
def setup():
    global HOME, db_conn, browser_bus
    HOME = getenv('HOME')
    if not HOME:
        logging.error('No HOME variable')
        exit(1) # Alternatively, we can raise an Exception using  a custom message, or we can create a new class that extends Exception.
```

Reply:
Yeah, I agree that it is redundant to include a default value. On most situations, HOME should always be set. It's okay to exit immediately of it wasn't set.

Reply:
Alright, i made the change with the if condition, that should resolve that file for these purposes.